### DISCUSSION TITLE
Custom model and latency

### DISCUSSION THREAD

**Discussion by davide445** [deepgram/community#555] [General help]
For a project I'm interested into the possibility to fine tune STT using a local German dialect for realtime streaming.
Wanted to kindly ask if this possibility is available for all the models, what kind and amount of data are needed (I suppose audio and word by word transcriptions, but both of them or also only audio), if this can be a specific licensing feature to attach to other licenses, if we can eventually self-host the model to lower the latency for local applications, or if there are endpoints based in south EU for the same reason.

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment]
Hi @davide445, fine-tuning models (or as we call them, "custom models") are available for enterprise customers with a cost of $10k USD. We would fine-tune an existing German model of ours with approximately 10-30 hours of audio data provided by you. You'd also provide a written list of particularly important keywords with their correct spellings (such as local slang, place names, or other proper names). We handle all transcription/annotation of the audio data you provide for training. You can also look over a [product sheet here](https://offers.deepgram.com/hubfs/Deepgram-Custom-Model-Training-Product%20Sheet.pdf?hsLang=en).

You mentioned interest in self-hosting the model - yes, you an absolutely do that with a fine-tuned model. You would use our "[on-prem](https://developers.deepgram.com/docs/on-prem-introduction)" (self-hosted) option. That would give you the ability to run the model on a server in a location of your choosing (such as EU or specifically Germany). That could be on a bare-metal server you own, or through any cloud provider (including common providers such as AWS, GCP, and Azure). Our hosted API is backed by servers in the US, so on-prem is a popular option for non-US customers with data sovereignty requirements.

I'm happy to answer any other questions, and let me know if you'd like me to connect you with our sales team for further discussion.
